### PR TITLE
add ACL config warning

### PR DESCRIPTION
**Pull request by blakadder**
## Proposed change
many users ran across this issue, I don't have experience with the addon so I'm just relaying what was submitted to our docs

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/tasmota.markdown [hunk 1]
```diff
@@ -52,3 +52,7 @@ Tasmota Buttons, Fans, Lights, relays, Sensors, Shutters and Switches are suppor
 You must also configure each Tasmota device's MQTT settings to communicate with whatever MQTT broker you are using. Enter the broker address under host and enter a username/password combination that allows access to the broker.
 
 If using the Mosquitto Broker add-on for Home Assistant, first create a new Home Assistant user under {% my users title="Configuration > Users" %}. Then, in Tasmota > Configure MQTT under "Host" enter the address of your Home Assistant instance and under "User" and "Password" enter the Home Assistant user you just created.
+
+Add the following to the ACL file (user section or general section): `topic write tasmota/discovery/#` to give Tasmota devices required write access to the `tasmota/discovery/#` topic.
```
Comment (frenck) [inline source/_integrations/tasmota.markdown:56]: Is specific for a certain installation method? The line above it, talks about the add-on (for which this doesn't apply/is needed and therefore confusing maybe?)

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/tasmota.markdown (hunk 1 above)
Comment (blakadder) [reply source/_integrations/tasmota.markdown:56]: I refers to the addon which is mentioned in the previous paragraph.

I only know many people complained its not working when using Home Assistant OS and noone yet bothered to add to documentation. I don't use HAOS and therefore the MQTT add-on

Comment (frenck) [reply source/_integrations/tasmota.markdown:56]: The add-on doesn't use an ACL by default and thus has write access by default; therefore I think this is incorrect and confusing.

Comment (blakadder) [reply source/_integrations/tasmota.markdown:56]: Fine by me then

**Pull request closed without merging**